lib/workers/repository/finalise/index.ts

```diff
@@ -13,4 +14,5 @@ export async function finaliseRepo(
   await platform.ensureIssueClosing(
     `Action Required: Fix Renovate Configuration`
   );
+  await clearRenovateRefs();
```

Comment:
It's not necessary to check `platformCommit`. The cleanup will work only if there are some refs pushed current run. As we don't push for now, the function body actually won't be executed in production.